js-apps/meep-frontend/src/js/containers/exec/dashboard-cfg-detail-pane.js

```diff
@@ -145,6 +149,104 @@ class DashCfgDetailPane extends Component {
     this.onUpdateDashCfg(DASH_CFG_START_TIME, '', null);
   }
 
+  // Callback function for postSeqQuery
+  postSeqQueryCb(error, data, format) {
+    if (error !== null) {
+      this.refreshSeqMetrics = true;
+      return;
+    }
+    var seqMetrics = (data && data.seqMetricString) ? data.seqMetricString : [];
+
+    // Nothing to do if no new metrics
+    if (seqMetrics.length === 0) {
+      return;
+    }
+    var seqChart = '';
+    // Add scenario-configured participants
+    var participants = getDashCfgFieldVal(this.viewCfg, DASH_CFG_PARTICIPANTS);
+    var dashParticipants = _.split(participants, ',');
+    _.forEach(dashParticipants, participant => {
+      seqChart += ('participant ' + participant + '\n');
```

Comment:
Should use another variable name here for participants that can be appended to the seqChart if necessary.

Reply:
Addressed.